src/libutil/util.hh

```diff
@@ -552,7 +552,22 @@ constexpr char treeNull[] = "    ";
 
 /* Determine whether ANSI escape sequences are appropriate for the
    present output. */
-bool shouldANSI();
+typedef enum {
+    colorDefault,
+    colorAlways,
+    colorAuto,
+    colorNever,
+} ColorMode;
+
+/* Whether or not to emit SGR-sequences (\e[…m)*/
+extern bool allowSelectGraphicRendition;
```

Comment:
```suggestion
extern bool allowSelectGraphicRendition;
```

This should probably be `allowANSI` or whatever. Nobody knows what "select graphic rendition" means.

Reply:
I am against calling it `allowANSI`, because another ANSI escape sequence are actually used and not filtered by `filterANSIEscapes`. To call it `allowColors` is not true either, because it is not just 30–39, 40–39, …, even though the command line flag is `--color`. Maybe `allowStyledOutput` is the closest? But maybe we should have a static variable for both stdout and stderr anyway and should rather create a `Output`-struct, place it there and use it for `stdout`/`stderr` in the `Logger`-class? I called it `allowSelectGraphicRendition`, because that is technically the most true and googleable, even though somewhat cryptic.
Are you fine with `allowStyledOutput`?